gui/guihtml/src/TGHtmlDraw.cxx

```diff
@@ -146,7 +146,7 @@ static void GetLetterIndex(char *zBuf, int index, int isUpper)
       zBuf[2] = 0;
    }
 
-   strcat(zBuf, ".");
+   strlcat(zBuf, ".", 2);
```

Comment:
Are you sure here is `2`? It must be total size of `zBuf`.  It fail already if zBuf already contains two character.